modules/oxide-components/src/main/ts/keynav/keyboard/FlowType.ts

```
@@ -0,0 +1,104 @@
+  const findCurrent = (component: SugarElement<HTMLElement>, flowConfig: FullFlowConfig): Optional<SugarElement<HTMLElement>> => {
+    // TODO: Figure out why we need this, without it enter inside a nested element would navigate up to the parent element if both parent and child have handlers
+    if (flowConfig.closest) {
+      return flowConfig.focusManager.get(component).bind((elem) => SelectorFind.closest(elem, flowConfig.selector));
+    } else {
+      return flowConfig.focusManager.get(component);
+    }
+  };
+
+  const execute = (component: SugarElement<HTMLElement>, _event: Event, flowConfig: FullFlowConfig) => findCurrent(component, flowConfig).bind((focused) => flowConfig.execute(focused));
+
+  const escape = (component: SugarElement<HTMLElement>, _event: Event, flowConfig: FullFlowConfig) => findCurrent(component, flowConfig).bind((focused) => flowConfig.escape(focused));
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)